src/types/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Enums } from '../enums'
+import { AccountID } from './account-id'
+import { Amount } from './amount'
+import { Blob } from './blob'
+const Field = Enums.Field
+const { Currency } = require('./currency')
+const { Hash128 } = require('./hash-128')
+const { Hash160 } = require('./hash-160')
+const { Hash256 } = require('./hash-256')
+const { PathSet } = require('./path-set')
+const { STArray } = require('./st-array')
+const { STObject } = require('./st-object')
+const { UInt16 } = require('./uint-16')
+const { UInt32 } = require('./uint-32')
+const { UInt64 } = require('./uint-64')
+const { UInt8 } = require('./uint-8')
+const { Vector256 } = require('./vector-256')
+
+const coreTypes = {
```

Comment:
Note that import and exports in other files are written as `import { CoreTypes } ...` and then `CoreTypes.Blob`. This was changed when imports were modified.